eixampledb-core/src/main/java/com/eixampledb/core/impl/NumberUtils.java

```diff
@@ -0,0 +1,43 @@
+package com.eixampledb.core.impl;
+
+import com.eixampledb.core.api.OperationValueTypeException;
+
+public class NumberUtils {
+
+    private NumberUtils() {}
+
+    public static Number parse(String value) {
```

Comment:
There is some boring cast and if-case that we moved here to this utils class so this code does not add noise to  implementations of EixampleDb interfaces.